test/RoleTest.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Net;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using static Conjur.Test.WebMocker;
+using System.Text;
+using System.Linq;
+
+namespace Conjur.Test
+{
+    public class RoleTest : Base
+    {
+        private string baseUrl = "test:///authz/test-account/roles/group";
+        
+        public RoleTest()
+        {
+            Client.Authenticator = new MockAuthenticator();
+        }
+
+        [Test]
+        public void TestRoleDoesExist()
+        {
+            ClearMocker();
+            Mocker.Mock(new Uri($"{baseUrl}/groupName"), "");
```

Comment:
Refactor it as constant? and return String.Empty?

Reply:
Fixed string.empty. Dealing with consts in UT feels me unnecessary.